pkg/acsengine/defaults.go

```diff
@@ -720,7 +720,7 @@ func setHostedMasterProfileDefaults(a *api.Properties) {
 
 func setDefaultCerts(a *api.Properties) (bool, error) {
 	if a.MasterProfile != nil && a.OrchestratorProfile.OrchestratorType == api.OpenShift {
-		return certgen.OpenShiftSetDefaultCerts(a, DefaultOpenshiftOrchestratorName, GenerateClusterID(a))
+		return certgen.OpenShiftSetDefaultCerts(a, api.DefaultOpenshiftOrchestratorName, a.GenerateClusterID())
```

Comment:
nit: `GenerateClusterID` seems to suggest to generate every time it is called... `GetClusterID` seems better?

Reply:
Certainly!

Reply:
ok.. maybe it's generated after all...

Reply:
Just to clarify, the existing `GenerateClusterID` (before this change) also generates w/ every invocation as well, right?

Reply:
Talked to @tariq1890 , it's seeded using the hash. so clusterID is always the same regardless how many times it is called

Reply:
In this case, I'd like to get `GetClusterID` :)

Reply:
@jackfrancis no, I've added a new field in Properties to store the ClusterID after it's generated during the first time.